src/config.ts

```diff
@@ -906,7 +906,7 @@ export default {
    * SourceBuffer is currently updating.
    * @type {Number}
    */
-  SOURCE_BUFFER_FLUSHING_INTERVAL: 2000,
+  SOURCE_BUFFER_FLUSHING_INTERVAL: 500,
```

Comment:
Why do you change this value ?

Reply:
Because here an "urgent" quality switch depend on the current SourceBuffer operations to be finished.

It is thus important that such operations terminate as soon as possible.